src/main/java/com/prgrms/needit/domain/user/openAPI/dto/BusinessesRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.prgrms.needit.domain.user.openAPI.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+public class BusinessesRequest {
+
+	@JsonProperty("businesses")
+	private List<BusinessRequest> businesses;
+
+	public BusinessesRequest(BusinessRequest business) {
+		List<BusinessRequest> request = new ArrayList<>();
+		request.add(business);
+		this.businesses = request;
+	}
```

Comment:
```suggestion
	@JsonProperty("businesses")
	private List<BusinessRequest> businesses = new ArrayList<>();

	public BusinessesRequest(BusinessRequest business) {
		businesses.add(business);
	}
```
이렇게 코드 작성하면 되지 않을까요? 중간에 리스트를 둔 이유가 궁금합니닷

Reply:
오홍 수정해볼게요!!